java/http/vertx/java-http-vertx-consumer/src/main/java/ConsumerApp.java

```diff
@@ -3,30 +3,31 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-import java.util.Map;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.TimeUnit;
-
-import io.jaegertracing.Configuration;
-import io.opentracing.Tracer;
-import io.opentracing.util.GlobalTracer;
-
 import io.vertx.config.ConfigRetriever;
 import io.vertx.config.ConfigRetrieverOptions;
 import io.vertx.config.ConfigStoreOptions;
 import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
 import io.vertx.core.json.JsonObject;
-import org.apache.logging.log4j.Logger;
+import io.vertx.tracing.opentelemetry.OpenTelemetryOptions;
 import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 public final class ConsumerApp {
 
     private static final Logger log = LogManager.getLogger(ConsumerApp.class);
-    
+
     private static String deploymentId;
 
     public static void main(String[] args) throws Exception {
-        Vertx vertx = Vertx.vertx();
+
+        VertxOptions vertxOptions = new VertxOptions();
+        vertxOptions.setTracingOptions(new OpenTelemetryOptions());
+        Vertx vertx = Vertx.vertx(vertxOptions);
```

Comment:
Actually we should configure the Vert.x instance with the OpenTelemetry options only if the tracing system is set by the user and it's opentelemetry.
With what we have now, we are enabling tracing in any case which is not right.

Reply:
TracingSystem deployed so tracing is only  enabled of `openTelemetry` is used with `TracingSystem` envVar.